activerecord/test/cases/tasks/postgresql_rake_test.rb

```diff
@@ -195,21 +195,56 @@ def setup
         'adapter'  => 'postgresql',
         'database' => 'my-app-db'
       }
+      @filename = "awesome-file.sql"
 
+      ActiveRecord::Base.dump_schemas = :schema_search_path
```

Comment:
We better not have side-effects in the `setup` method. I'd rather modify the global state in an individual test and have that one be responsible for resetting it.